### PR TITLE
Update PhotoAdapter.java

### PR DESCRIPTION
**Pull request by Aspsine**
Fix NullPointException when phone has no navigation bar click image.
#2

### CODE REVIEW

**Comment by Aspsine** [comment]
I signed it!

File: app/src/main/java/com/example/android/unsplash/ui/grid/PhotoAdapter.java [hunk 1]
```diff
@@ -84,11 +84,17 @@ public void onClick(View v) {
                 View navBackground = decorView.findViewById(android.R.id.navigationBarBackground);
                 Pair statusPair = Pair.create(statusBackground,
                         statusBackground.getTransitionName());
-                Pair navPair = Pair.create(navBackground, navBackground.getTransitionName());
-
-                host.startActivity(intent,
-                        ActivityOptions.makeSceneTransitionAnimation(host,
-                                authorPair, photoPair, statusPair, navPair).toBundle());
+                ActivityOptions options;
+                // some phone doesn't has a 'navigation bar' so we do some extra check
```
Comment (keyboardsurfer) [inline app/src/main/java/com/example/android/unsplash/ui/grid/PhotoAdapter.java]: Please change to:
"Some devices don't have a navigation bar, so we have to check if it's available."

File: app/src/main/java/com/example/android/unsplash/ui/grid/PhotoAdapter.java [hunk 2]
```diff
@@ -84,11 +84,17 @@ public void onClick(View v) {
...
+                if (navBackground != null) {
```
Comment (keyboardsurfer) [inline app/src/main/java/com/example/android/unsplash/ui/grid/PhotoAdapter.java]: Invert the condition and resulting code.

File: app/src/main/java/com/example/android/unsplash/ui/grid/PhotoAdapter.java [hunk 3]
```diff
@@ -84,11 +84,17 @@ public void onClick(View v) {
...
+                    Pair navPair = Pair.create(navBackground, navBackground.getTransitionName());
+                    options = ActivityOptions.makeSceneTransitionAnimation(host,
+                            authorPair, photoPair, statusPair, navPair);
+                }else {
```
Comment (keyboardsurfer) [inline app/src/main/java/com/example/android/unsplash/ui/grid/PhotoAdapter.java]: missing space

**Pull request merged**